common/model/label-field.js

```diff
@@ -3,7 +3,7 @@ import type {HTMLString} from '../services/prismic/types';
 import type {ContentFormatId} from './content-format-id';
 
 export type LabelField = {|
-  id: ?ContentFormatId,
+  id?: ?ContentFormatId,
```

Comment:
When would we not have an id?

Reply:
For instances where we use labels for things that aren't content formats? It was the most straightforward fix I had for Flow, but perhaps I should be adding null IDs to those cases?

Reply:
We should always have an ID as it comes from Prismic (it's synced all the way to `label-field` in `prismic-model`).
Where is it breaking? Where we send it down from the list?